ck.sh

```diff
@@ -58,36 +58,35 @@ case "$COMMAND" in
 
 		case $yn in
 			[Yy]*)
-				# Copy custom skin to ckeditor-dev
-				cp -r skins/moono-lexicon ckeditor-dev/skins/moono-lexicon
-
-				# Copy custom plugins to ckeditor-dev
-				cp -r plugins ckeditor-dev
-
-				# Copy skin files for plugins
-				pluginsWithSkins=$(find ckeditor-dev/plugins -maxdepth 2 -mindepth 2 -type d -name skins)
-				for plugin in $pluginsWithSkins; do
-					pluginDir=$(dirname "$plugin")
-					pluginName=$(basename "$pluginDir")
-					pluginSkinPath="skins/moono-lexicon/plugins/$pluginName/skins/moono-lexicon"
-					if [[ -d "$pluginSkinPath" ]]; then
-						cp -r "$pluginSkinPath" "$pluginDir/skins"
-					fi
-				done
+				# Clean and make a temporal copy of ckeditor-dev to work with
```

Comment:
"temporal" means "relating to time"; I think you want "temporary" here; same on 107.

Reply:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia0.giphy.com%2Fmedia%2FweNyGjdcQ3d9S%2Fgiphy.gif&token=temporal" data-gitmeme-token="temporal"><img src="https://media0.giphy.com/media/weNyGjdcQ3d9S/giphy.gif" title="Created by gitme.me with /temporal"/></a>